subprojects/gradle/instrumentation-tests/src/test/kotlin/com/avito/instrumentation/InstrumentationTestsTaskVerdictSerializationTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.avito.instrumentation
+
+import com.avito.instrumentation.internal.InstrumentationTestsActionFactory
+import com.avito.instrumentation.internal.verdict.InstrumentationTestsTaskVerdict
+import com.github.salomonbrys.kotson.fromJson
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.Test
+
+public class InstrumentationTestsTaskVerdictSerializationTest {
```

Comment:
```suggestion
internal class InstrumentationTestsTaskVerdictSerializationTest {
```

Reply:
Does it affect something?

Reply:
For the sake of consistency.
We have 128 internal tests and only 2 public.
I'd like to choose one approach and don't think about it.